crates/yas_core/src/tests/test_contracts/test_yas_pool.cairo

```diff
@@ -486,81 +491,1014 @@ mod YASPoolTests {
+                            100
+                        );
+                    mint_callback
+                        .mint(
+                            yas_pool.contract_address,
+                            WALLET(),
+                            IntegerTrait::<i32>::new(240, true),
+                            IntegerTrait::<i32>::new(0, false),
+                            40
+                        );
+                    //burn
+                    assert(1 == 2, 'burn() func doesnt exist yet');
+
+                    assert(
+                        yas_pool.ticks(IntegerTrait::<i32>::new(240, true)).liquidity_gross == 500,
+                        'wrong liquidity_gross amount'
+                    );
+                    assert(
+                        yas_pool.ticks(IntegerTrait::<i32>::new(0, false)).liquidity_gross == 50,
+                        'wrong liquidity_gross amount 2'
+                    );
+                }
+
+
+                // TODO: 'clears tick lower if last position is removed'
+                //#[test]
+                //#[available_gas(200000000)]
+                fn test_clear_tick_lower() {
+                    //to implement:
+                    //await mint(wallet.address, -240, 0, 100) -done
+                    //await pool.burn(-240, 0, 100)
+                    //const { liquidityGross, feeGrowthOutside0X128, feeGrowthOutside1X128 } = await pool.ticks(-240) -done
+                    //expect(liquidityGross).to.eq(0) -done
+                    //expect(feeGrowthOutside0X128).to.eq(0) -done
+                    //expect(feeGrowthOutside1X128).to.eq(0) -done
+
+                    let (yas_pool, token_0, token_1, mint_callback, min_tick, max_tick) = setup();
+                    mint_callback
+                        .mint(
+                            yas_pool.contract_address,
+                            WALLET(),
+                            IntegerTrait::<i32>::new(240, true),
+                            IntegerTrait::<i32>::new(0, false),
+                            100
+                        );
+                    //burn
+                    assert(1 == 2, 'burn() func doesnt exist yet');
+
+                    let tick_info = yas_pool.ticks(IntegerTrait::<i32>::new(240, true));
+                    assert(tick_info.liquidity_gross == 0, 'wrong liquidity_gross amount');
+                    assert(
+                        tick_info.fee_growth_outside_0X128 == 0, 'wrong fee_growth_outside_0X128'
+                    );
+                    assert(
+                        tick_info.fee_growth_outside_1X128 == 0, 'wrong fee_growth_outside_1X128'
+                    );
+
+                    assert(1 == 2, 'burn() func doesnt exist yet');
+                }
+
+                // TODO: 'clears tick upper if last position is removed'
+                //#[test]
+                //#[available_gas(200000000)]
+                fn test_clear_tick_upper() {
```

Comment:
Please comment this function with the proper message explaining that we are waiting for the burn function to be merged.